__tests__/app.test.js

```diff
@@ -191,3 +191,28 @@ describe("/api/users", () => {
       });
   });
 });
+
+describe("GET /api/articles", () => {
+  test("status 200 return article array ", () => {
+    return request(app)
+      .get("/api/articles")
+      .expect(200)
+      .then(({ body: { articles } }) => {
+        expect(articles).toHaveLength(12);
+        expect(articles).toEqual(
+          expect.arrayContaining([
+            expect.objectContaining({
+              article_id: expect.any(Number),
+              comment_count: expect.any(Number),
+              title: expect.any(String),
+              topic: expect.any(String),
+              author: expect.any(String),
+              body: expect.any(String),
```

Comment:
no `body` key is required in this response, if you think about it from a user POV, getting the body from every article is gonna be a massive amount of information to see at once - by omitting it we make our lives easier as users (if they want to see the body they can select by article id)